features/meter.yml

```
@@ -1,8 +1,9 @@
-draft_date: 2024-06-05
 name: <meter>
-description: The `<meter>` element TODO.
+description: The `<meter>` element represents either a scalar value within a known range or a fractional value, and is typically rendered as a progress bar.
```

Copy editing for brevity

```suggestion
description: The `<meter>` element represents a scalar value within a known range or a fractional value, typically shown as a progress bar.
```